docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -16,30 +16,140 @@ Features
 New in Version 5.0.5
 --------------------
 This is a point release which fixes some issues reported in earlier versions
-of 5.0.x:
-
-* The issue with the optimiser not respecting parameter bounds where set has been
-  resolved by the move to a later version of the Bumps package.
-* ?
-
-There are also some new features in this version:
-
-* ?
-
-This version of SasView is built with Sasmodels 1.0.6 and Bumps 0.9.0. 
-
-New features/improvements
-^^^^^^^^^^^^^^^^^^^^^^^^^
-* ?
-
-
-Bug fixes
-^^^^^^^^^
-* ?
+of 5.0.x. A few highlights are:
+
+* The long standing issue with the Levenberg-Marquardt optimiser not respecting
+  parameter bounds where set has been resolved by the move to a later version of
+  the Bumps package.
```

Comment:
I am sure it is correct but this part "parameter bounds where set has been" reads bit strange. Can we potentialy skip "where set" or does it change the meaning?